dwave/optimization/mathematical.py

```diff
@@ -97,6 +101,35 @@ def add(x1: ArraySymbol, x2: ArraySymbol, *xi: ArraySymbol) -> typing.Union[Add,
     raise RuntimeError("implementated by the op() decorator")
 
 
+def logical(x: ArraySymbol) -> Logical:
+    r"""Return the element-wise truth value on the given symbol.
```

Comment:
Maybe worth reiterating here that all values but 0 are truthy?

Reply:
Sure, I can make that explicit.